codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/customization/route53/Route53InvalidBatchErrorIntegration.kt

```diff
@@ -27,14 +27,11 @@ class Route53InvalidBatchErrorIntegration : SwiftIntegration {
     private val httpResponseBindingErrorGenerator = SectionWriter { writer, previousCode ->
         val operationErrorName = writer.getContext("operationErrorName") as String
         if (operationErrorName == "ChangeResourceRecordSetsOutputError") {
```

Comment:
I discovered that this Route 53 customization isn't currently working.

Wrote a ticket to get it under test & verified working:
https://github.com/awslabs/aws-sdk-swift/issues/1027